src/m1_facilitator/handlers/DeclaredDepositIntentsHandler.ts

```diff
@@ -169,4 +178,16 @@ export default class DeclaredDepositIntentsHandler extends ContractEntityHandler
       Logger.debug(`Deposit intent ${depositIntent} saved.`);
     }
   }
+
+  /**
+   * Checks if value token address needs to be facilitated.
+   *
+   * @param valueTokenAddress Value token address.
+   */
+  private isFacilitateToken(valueTokenAddress: string): boolean {
+    if (this.facilitateTokens.size === 0 || this.facilitateTokens.has(valueTokenAddress)) {
+      return true;
+    }
+    return false;
```

Comment:
Just return 
`return this.facilitateTokens.size === 0 || this.facilitateTokens.has(valueTokenAddress)`

Reply:
Done!